stable/insights-admission/templates/cert.yaml

```diff
@@ -3,52 +3,21 @@ apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: {{ include "insights-admission.fullname" . }}-cert
-  annotations:
-    "helm.sh/hook": pre-install
```

Comment:
the need for this to be installed before any resources was removed right?

Reply:
I'm not quite sure why it was here - my gut is that Bader was trying to control the order that these certs were installed.

Unfortunately, if you use `hook`, helm won't uninstall those resources when running `helm uninstall`, so I think it's best to remove it where possible. This installs and runs fine for me without it.

Will definitely want to get Bader's eyes on this when he comes back though to double check.